task/thermal/src/bsp/sidecar_bcd.rs

```diff
@@ -57,11 +57,12 @@ bitflags::bitflags! {
 
 #[allow(dead_code)]
 pub(crate) struct Bsp {
-    pub inputs: &'static [InputChannel],
-    pub dynamic_inputs: &'static [SensorId],
+    /// Controlled sensors
+    pub inputs: &'static [InputChannel; NUM_TEMPERATURE_INPUTS],
+    pub dynamic_inputs: &'static [SensorId; NUM_DYNAMIC_TEMPERATURE_INPUTS],
 
     /// Monitored sensors
-    pub misc_sensors: &'static [TemperatureSensor],
+    pub misc_sensors: &'static [TemperatureSensor; NUM_TEMPERATURE_SENSORS],
```

Comment:
I'm guessing this was for consistency even though we don't do anything with these sensors?

Reply:
Yeah, I wanted to switch to strongly-sized arrays throughout.